CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
 ## 14.1.4
 
 ### Features
+- `[lf-login]`: Add function `beforeFetchRequestAsync` to login component
+- `[lf-login]`: Add function `afterFetchResponseAsync` to login component
```

Comment:
need to add external interface change to lf-documentation
- needs to be added in lf-login-documentation.component.html
- needs to be added to CHANGELOG, version doesn't need to be updated bc you already rev-ed it in the strings PR